backend/config/settings/base.py

```diff
@@ -91,9 +91,8 @@
         "rest_framework.permissions.IsAuthenticatedOrReadOnly",
     ),
     "DEFAULT_AUTHENTICATION_CLASSES": (
-        "rest_framework_jwt.authentication.JSONWebTokenAuthentication",

```

Comment:
Why no more JWT?

Reply:
Good catch! I tried so many different configs, that I didn't remember that I deleted it. I put it back.
See commit #https://github.com/code-for-canada/project-thundercat/pull/308/commits/5c358f2235c97f6bdbdbacf918350f38996507dc